LondonBritishLibrary/orient/BLorient8823.xml

```diff
@@ -0,0 +1,266 @@
+                        <altIdentifier><idno>Parham no. 56</idno></altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <msItem xml:id="ms_i1">
+                            <locus from="3ra" to="22va"/>
+                            <title ref="LIT1340EnochE"/>                  
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                            <locus from="22va" to="74ra"/>
+                            <title ref="LIT1719Bookso"/>
+                                <msItem xml:id="ms_i2.1">
+                                    <locus from="22vra" to="35rb"/>
+                                    <title ref="LIT2697Sam"/>
+                                    <note>The text ends with the first verse of 2 Samuel, which is also repeated in its proper place.</note>
+                                    <explicit xml:lang="gez">ተፈጸመ፡ መጽሐፈ፡ ነገሥት፡ ፩ስብሐት፡ ለእግዚአብሔር፡ ዘኂሩት፡ ልማዱ። ለዓለመ፡ ዓለም፡ አሜን። ለቀዳማዊ፡ መጽሐፈ፡ 
+                                        ነገሥት፡ ወጠኖ፡ ሳሙኤል፡ ወፈጸምዎ፡ ጋድ፡ ወናታን።</explicit>
+                                </msItem>
+                                <msItem xml:id="ms_i2.2">
+                                    <locus from="35rb" to="46vb"/>
+                                    <title ref="LIT2698Sam"/>
+                                    <explicit xml:lang="gez">ለዳግማዊ፡ ነገሥት፡ ጸሐፍዎ፡ ናታን፡ ወጋድ።</explicit>
+                                </msItem>
+                                <msItem xml:id="ms_i2.3">
+                                    <locus from="46vb" to="60vb"/>
+                                    <title ref="LIT2699Kings"/>
+                                    <explicit xml:lang="gez">ተፈጸመ፡ ነገሥት፡ ፫።</explicit>
+                                </msItem>
+                                <msItem xml:id="ms_i2.4">
+                                    <locus from="60vb" to="74ra"/>
+                                    <title ref="LIT2700Kings"/>
+                                    <explicit xml:lang="gez">ተፈጸመ፡ በዝየ፡ ነገሥት፡ ፬ወስብሐት፡ ለእግዚአብሔር፡ ለዓለመ፡ ዓለም፡ አሜን።</explicit>
+                                    <note>Starting with <locus target="#71ra"/>, the hand changes and the chapters are numbered from 71 to 100. 
+                                        However, this numbering is not the usual one.</note>
+                                </msItem>
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                            <locus from="75ra" to="95ra"/>
+                            <title ref="LIT5841EthioMaccabBooks"/>   
+                            <incipit xml:lang="gez">በመንግሥተ፡ ሜዳናውያን፡ ወሞአባውያን። ወሀሎ፡ ፩ንጉሥ፡ ዘስሙ፡ ጺሩጻይዳን፡ ፈታሒሃ፡ ለእኪት፡ </incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i4">
+                            <locus from="95ra" to="103va"/>
+                            <title>Three Compositions for Moral Edification.</title>
```

Comment:
I also could not find a record for this work through searching for the text quoted here. I think it would warrant an issue and possibly creation of a new work ID

Reply:
I created an Issue: https://github.com/BetaMasaheft/Documentation/issues/2482